php-nginx/entrypoint.sh

```diff
@@ -138,4 +134,7 @@ chmod -R 550 ${DOCUMENT_ROOT}
 # Change the www-data's shell back to /usr/sbin/nologin
 chsh -s /usr/sbin/nologin www-data
 
+# Remove the loose php-cli.ini
+rm ${PHP_DIR}/lib/php-cli.ini
```

Comment:
We can set `display_errors=STDERR`, which will fix this issue. I'm looking into it to see if we want to set it for web as well, since STDERR should just be logged.  I verified this locally:
```php
// test.php
<?php
exec('php -r "phpinfo();"', $output, $return_var);
var_dump($output, $return_var);
// array(0) { } int(0)
```
When accessed through a web server, the CLI error message is not displayed, only if it's run on the CLI (through `stderr`)